Code/GraphMol/SmilesParse/SmilesParse.h

```diff
@@ -11,13 +11,85 @@
 #ifndef RD_SMILESPARSE_H
 #define RD_SMILESPARSE_H
 
-#include <GraphMol/RWMol.h>
 #include <GraphMol/SanitException.h>
 #include <string>
 #include <exception>
 #include <map>
+#include <memory>
 
 namespace RDKit {
+class RWMol;
+class Atom;
+class Bond;
+
+namespace SmilesParse {
+class RDKIT_SMILESPARSE_EXPORT SmilesParseException : public std::exception {
+ public:
+  SmilesParseException(const char *msg) : _msg(msg) {}
+  SmilesParseException(const std::string msg) : _msg(msg) {}
+  const char *what() const noexcept override { return _msg.c_str(); }
+  ~SmilesParseException() noexcept override = default;
+
+ private:
+  std::string _msg;
+};
+
+}  // namespace SmilesParse
+
+namespace v2 {
+namespace SmilesParse {
+using RDKit::SmilesParse::SmilesParseException;
+
+struct RDKIT_SMILESPARSE_EXPORT SmilesParserParams {
+  int debugParse = 0;   /**< enable debugging in the SMILES parser*/
```

Comment:
I don't see any differences between this and the original SmilesParserParams.  Am I missing something?